astropy/stats/tests/test_funcs.py

```diff
@@ -440,6 +441,36 @@ def test_mad_std():
         data = normal(5, 2, size=(100, 100))
         assert_allclose(funcs.mad_std(data), 2.0, rtol=0.05)
 
+def test_mad_std_scalar_return(recwarn):
```

Comment:
Instead of `recwarn` most tests use `astropy.utils.catch_warnings` as context manager. The `recwarn` lead to some failures on python 3.6.